projects/golang/go/VulnerabilityManagement/eks-distro-golang-vex.json

```diff
@@ -0,0 +1,101 @@
+{
+  "document": {
+    "category": "csaf_vex",
+    "csaf_version": "2.0",
+    "notes": [
+      {
+        "category": "summary",
+        "text": "VEX Document for AWS EKS Distribution Golang.",
+        "title": "EKS Golnag VEX"
+      }
+    ],
+    "publisher": {
+      "category": "vendor",
+      "name": "Amazon Web Services Inc",
+      "namespace": "https://aws.amazon.com/"
+    },
+    "title": "Vulnerability Tracking Document for AWS EKS Distro Golang",
+    "tracking": {
+      "current_release_date": "2023-08-28T11:00:00Z",
+      "id": "2023-08-EKS-D-GO",
```

Comment:
What's the significance of this string?

Reply:
tbh i didn't really think about it; i just made the ID yyyy-dd and then our own identifier. suggestions welcome